geonode_mapstore_client/client/themes/geonode/scss/_filter-form.scss

```diff
@@ -28,14 +27,31 @@
         flex: 1;
     }
     .gn-filter-form-footer {
+        width: 20.3em;
         display: flex;
         flex-direction: row-reverse;
         align-items: center;
-        margin-bottom: 0.5rem;
-        @media screen and (max-width: 768px) {
-            margin-top: 5rem;
+        margin-bottom: 1.5rem;
+        position: fixed;
```

Comment:
we should not use position fixed for the footer if possible, see my general comment